R/add_to_view.R

```diff
@@ -46,36 +56,46 @@ add_to_view <- function(object, call.args, side){
 #'                error_bar(y.high=1))
 #' gs
 #' 
-#' gsplot:::call_arguments('points', x=2:6, y=2:6, ylim=c(-1, 11))
-#' gsplot:::call_arguments('points', x=1:5, y=1:5, xlim=c(0,10), ylim=c(0,10), 
-#'                callouts(labels=c(rep(NA, 4), "oh")))
+#' gsplot:::filter_arguments('points', x=2:6, y=2:6, ylim=c(-1, 11))$call.args
+#' gsplot:::filter_arguments('points', x=1:5, y=1:5, xlim=c(0,10), ylim=c(0,10), 
+#'                callouts(labels=c(rep(NA, 4), "oh")))$extracted.args
 #' @keywords internal
-call_arguments <- function(fun.name, ...){
+filter_arguments <- function(fun.name, ...){
```

Comment:
since some functions use only call.args or maybe ignore embedded args, I would also like to think about adding an additional arg as `out=c("call.args","option.args","extracted.args")` by default. Thoughts? So `bgCol` would use `out=c("call.args")`